cypress/integration/plugins/anomaly-detection-dashboards-plugin/overview_spec.js

```diff
@@ -39,7 +39,7 @@ context('Overview page', () => {
   // it('Empty dashboard redirects to overview page', () => {
```

Comment:
not related to this PR, but why there are commented code? could we clean up and remove unused code. or fix flake test if we still need?

Reply:
link to a AD issue https://github.com/opensearch-project/opensearch-dashboards-functional-test/issues/278 will share such as well.